src/components/styles/MainContentFlexContainer.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
+
+const MainContentFlexContainer = styled.div`
+  display: flex;
```

Comment:
In case you didn't know you can base styled components off existing ones if you want:

```
const MainContentFlexContainer = styled(MainContentContainer)`
  display: flex;
`
```
MainContentFlexContainer inherits all its parents style properties and can add/override its own. Any changes to it's parent will also affect this component.